packages/components-js/projects/partials/scripts/generateInitialStylesPartial.ts

```diff
@@ -102,6 +107,17 @@ const slottedStyles: Styles = {
   },
 };
 
+const siblingStyles: Styles = {
+  '@global': {
+    '%%tagNamesWithSiblingTabindex%%': {
+      '& ~ [tabindex="0"][role="tabpanel"]': addImportantToEachRule({
```

Comment:
do we need the `~`?

Reply:
Hi @denyo , the `~` is needed to select all siblings of pervious element, e.g. `p-tabs-bar`:
<img width="675" alt="Bildschirmfoto 2023-05-03 um 10 06 35" src="https://user-images.githubusercontent.com/40788023/235862562-0ea1261c-6519-4f13-af03-5dc406f6ce34.png">